Plots/CB_Height.py

```diff
@@ -0,0 +1,240 @@
+    cmap=newcmp,
+    cbar_kwargs={
+        "orientation": "vertical",
+        "ticks": np.arange(100, 1600, 100),
+        "label": "",
+        "shrink": 0.7,})
+
+# Use geocat.viz.util convenience function to set axes parameters without calling several matplotlib functions
+# Set axes limits, and tick values
+gvutil.set_axes_limits_and_ticks(
+    ax2,
+    xlim=(0, 90),
+    ylim=(0, 90),
+    xticks=np.linspace(-180, 180, 13),
+    yticks=np.linspace(-90, 90, 7))
+
+# Use geocat.viz.util convenience function to make plots look like NCL plots by using latitude, longitude tick labels
+gvutil.add_lat_lon_ticklabels(ax2)
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax2, labelsize=12)
+
+# Use geocat.viz.util convenience function to set titles and labels without calling several matplotlib functions
+gvutil.set_titles_and_labels(
+    ax2,
+    maintitle="Magma Color Projection \n of PBL Height",
+    maintitlefontsize=16,
+    xlabel="",
+    ylabel="")
+
+###############################################################################
+#Plot third color map:
+
+ax3 = plt.subplot(2, 2, 3, projection=projection)
+ax3.coastlines(linewidths=0.5)
+ax3.add_feature(cfeature.LAND, facecolor="lightgray")
+plt.subplots_adjust(wspace=0.5)
+# Import an NCL colormap
+newcmp = "coolwarm"
+
+# Contourf-plot data
+heatmap = t.plot.contourf(
+    ax=ax3,
+    transform=projection,
+    levels=40,
+    vmin=100,
+    vmax=1600,
+    cmap=newcmp,
+    cbar_kwargs={
+        "orientation": "vertical",
+        "ticks": np.arange(100, 1600, 100),
+        "label": "",
+        "shrink": 0.7,})
+
+# Use geocat.viz.util convenience function to set axes parameters without calling several matplotlib functions
+# Set axes limits, and tick values
+gvutil.set_axes_limits_and_ticks(
+    ax3,
+    xlim=(0, 90),
+    ylim=(0, 90),
+    xticks=np.linspace(-180, 180, 13),
+    yticks=np.linspace(-90, 90, 7))
+
+# Use geocat.viz.util convenience function to make plots look like NCL plots by using latitude, longitude tick labels
+gvutil.add_lat_lon_ticklabels(ax3)
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax3, labelsize=12)
+
+# Use geocat.viz.util convenience function to set titles and labels without calling several matplotlib functions
+gvutil.set_titles_and_labels(
+    ax3,
+    maintitle="Coolwarm Diverging Color \n Projection of PBL Height",
+    maintitlefontsize=16,
+    xlabel="",
+    ylabel="")
+
+###############################################################################
+#Plot fourth color map:
+
+ax4 = plt.subplot(2, 2, 4, projection=projection)
+ax4.coastlines(linewidths=0.5)
+# Change land colors from gray to green to avoid confusion
+ax4.add_feature(cfeature.LAND, facecolor="lightgray")
+plt.subplots_adjust(wspace=0.5)
+# Import an NCL colormap
+newcmp = "Reds"
+
+# Contourf-plot data
+heatmap = t.plot.contourf(
```

Comment:
There seems to be a lot of function calls with the same/similar arguments that you reuse again many times like contourf and set_axes_limits_and_ticks. Not sure if there would be a way to store these functions and arguments in a variable or helper function, but that would probably reduce program length.